docs/site/downloads/cldr-46.md

```diff
@@ -6,8 +6,8 @@ title: CLDR 46 Release Note
 
 | No. |    Date    | Rel. Note |  Data  |  Charts  | Spec |   Delta  | GitHub Tag | Delta DTD | CLDR JSON |
 |:---:|:----------:|:---------:|:------:|:--------:|:------------:|:---:|:----------:|:---------:|:---------:|
+|  46.1 | 2024-12-18 | [v46.1](/index/downloads/cldr-46.1) | [CLDR46.1](https://unicode.org/Public/cldr/46.1/) | [Charts46.1](https://unicode.org/cldr/charts/46.1/) | [LDML46](https://www.unicode.org/reports/tr35/tr35-73/tr35.html) | [Δ46.1](https://unicode-org.atlassian.net/issues/?jql=project+%3D+CLDR+AND+status+%3D+Done+AND+resolution+%3D+Fixed+AND+fixVersion+%3D+%2246.1%22+ORDER+BY+priority+DESC) | [release-46](https://github.com/unicode-org/cldr/releases/tag/release-46.1) | _n/a_ | [46.0.0](https://github.com/unicode-org/cldr-json/releases/tag/46.1.0) |
```

Comment:
index/downloads/cldr-46.1 is not the right link for a release note, we are just adding stuff into index/downloads/cldr-46. And are we planning to generate new charts for 46.1? And the spec will be version 74, not 73. And the release tag will be "release-46-1" (like the other dot releases), not 46.1

Reply:
And the entry for the json tag should have [46.1.0]

Reply:
Well, I will unblock this so we can merge and then make a separate PR to fix the issues noted above.